web_app/templates/home.html

```diff
@@ -29,29 +19,17 @@ <h3 class="text-center">Instructions:</h3>
                 <form action="{{ url_for('finder_routes.finder') }}" method="post">
                     <div class="form-group">
                         <label for="userCity">Enter your city:</label>
-                        <input type="text" class="form-control" id="userCity" placeholder="City">
+                        <input type="text" class="form-control" id="userCity" name="userCity" placeholder="New York">
```

Comment:
in order for the form to pass the data to the route, we need to add "name" attributes to each form input.